src/main/java/org/hl7/davinci/priorauth/ClaimEndpoint.java

```diff
@@ -200,8 +198,12 @@ private Bundle processBundle(Bundle bundle) {
     String id = UUID.randomUUID().toString();
 
     // get the patient
-    Claim claim = (Claim) bundle.getEntryFirstRep().getResource();
-    String patient = FhirUtils.getPatientIdFromResource(claim);
+    Claim claim = FhirUtils.getClaimFromRequestBundle(bundle);
+    String patient = FhirUtils.getPatientIdentifierFromBundle(bundle);
```

Comment:
I was a little confused by the method name getPatientIdentifierFromBundle; patient is a String.    My database training makes me think "numeric" whenever I see "Identifier" as a method or variable.

Reply:
`Identifier` is a FHIR property. I don't think we should change it. I think this against what you were saying but I actually think `patient` should be renamed to `patientIdentifier` since it is only storing the identifier and not the entire patient resource

Reply:
I think that ``patientIdentifier`` is clearer as to what we are  storing, rather than ``patient``.

Reply:
`patient` is used in a lot of places. While some of the changes were made here I created PRIORAUTH-63 a while back to make sure patient Identifier and priorauth identifier are used correctly. I will add renaming `patient` to `patientIdentifier` to this task